scripts/build-agent

```diff
@@ -2,6 +2,7 @@
 set -e
 
 source $(dirname $0)/version
+source $(dirname $0)/export-config
```

Comment:
Why are we sourcing here if this file does not use any variables?

Reply:
I might have added this here unnecessarily, yeah. Although for the future, this wouldn't hurt - if any variables will be used in the agent binary build...
Will remove if you think it's best.